fetch.bs

```diff
@@ -1196,7 +1196,8 @@ or "<code>worker</code>".
 
    <li>
     <p>If <var>read</var> is fulfilled with an object whose <code>done</code> property is false
-    and whose <code>value</code> property is a <code>Uint8Array</code> object, then run these steps:
+    and whose <code>value</code> property is a <code>Uint8Array</code> object, then run these steps
+    until fetch is <a for=fetch>terminated</a>:
```

Comment:
The problem with this setup is that you don't reach the later "Otherwise" if the fetch is terminated because the initial "If" conditional is already met. So you probably need to create a new block after "then" where inside you first run steps until terminated and then handle the termination if terminated.